rollup_health_and_sanity_metrics.py

```diff
@@ -331,13 +330,14 @@ def check_publishing_sla_for_device(device, window, sla):
         node_healthy = True
 
         for device in node.devices:
-            # the idea here is to translate the publishing frequency into a kind of SLA. here
-            # we're saying that after breaking the series up into window the size of the publishing
-            # frequency, we should see 1 sample per window in 90% of the windows.
-            healthy = check_publishing_sla_for_device(device, window, 0.90)
-            # accumulate full node health
-            node_healthy = node_healthy and healthy
-            add_device_health_check_record(node.vsn, device, healthy)
+            if device in device_output_table:
```

Comment:
Really minor ask - can you "flatten" this condition so it reads:

```
        for device in node.devices:
             if device not in device_output_table:
                 continue
             ... rest of logic...
```

I know it's only one conditional but I personally find it easier to read when less code nesting happens.